packages/insomnia-inso/src/openapi-2-kong.d.ts

```diff
@@ -0,0 +1,57 @@
+// TODO(TSCONVERSION) remove this when the conversion for o2k is complete
+
+declare module 'openapi-2-kong' {
+  /** TODO(TSCONVERSION) this can be taken from the official types OpenAPIV3.Document */
+  type OpenAPI3Document = {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [todo: string]: any;
+  }
+
+  export type ConversionResultType = 'kong-declarative-config' | 'kong-for-kubernetes'
+
+  export type DeclarativeConfigResult = {
+    type: 'kong-declarative-config',
+    label: string,
+    documents: Object,
+    warnings: {
+      severity: number,
+      message: string,
+      range: {
+        /* TODO */
+      },
+    }[],
+  };
+
+  export type KongForKubernetesResult = {
+    type: 'kong-for-kubernetes',
+    label: string,
+    documents: Object[],
+    warnings: {
+      severity: number,
+      message: string,
+      range: {
+        /* TODO */
```

Comment:
Do we need to fill this in?

Reply:
no, not now, but it'll come for free (see the comment above) when the o2k conversion happens anyway so as with the other one for send-request I tried not to spend a bunch of time reversing o2k in this PR.